app/services/selection.ts

```diff
@@ -648,19 +626,19 @@ export class Selection {
     this.getNodes().forEach(node => node.remove());
   }
 
-  nudgeActiveItemsLeft() {
+  nudgeLeft() {
```

Comment:
not sure if these are missed commands or not

Reply:
Nope.  These are lower level functions that are called by the command.

In a typical web app, you can think of our command layer sitting at the endpoint/controller level.  It's as close as possible to where actual requests are coming in from the user.  These functions are more at the model / business logic layer, if that makes sense.  I decided on this separation because I want us to be able to re-use logic as much as possible internally without accidentally pushing undo operations onto the stack.  This is why commands are almost exclusively (with a few exceptions) triggered from Vue components rather than services.